test/python/circuit/test_circuit_load_from_qasm.py

```diff
@@ -29,18 +30,16 @@ class LoadFromQasmTest(QiskitTestCase):
     def setUp(self):
         super().setUp()
         self.qasm_file_name = "entangled_registers.qasm"
-        self.qasm_dir = os.path.join(
-            os.path.dirname(os.path.dirname(os.path.abspath(__file__))), "qasm"
-        )
-        self.qasm_file_path = os.path.join(self.qasm_dir, self.qasm_file_name)
+        self.qasm_dir = (Path(__file__).parent.parent / "qasm").absolute()
```

Comment:
If you are going to use `pathlib`, you probably want to use `Path.parents[1]` rather than `Path.parent.parent`.